generator/mavschema.xsd

```diff
@@ -16,7 +16,17 @@
         </xs:restriction>
     </xs:simpleType>
 </xs:attribute>
+<xs:attribute name="increment" type="xs:float"/> <!-- param -->
 <xs:attribute name="index" type="xs:unsignedByte"/> <!-- param -->
+<xs:attribute name="isDestination" type="xs:boolean" default="true"/> <!-- param -->
+<xs:attribute name="hasLocation" type="xs:boolean" default="true"/> <!-- param -->
+<xs:attribute name="decimalPlaces" type="xs:unsignedByte"/> <!-- param -->
+<xs:attribute name="defaultValue" type="xs:float"/> <!-- param -->
+<xs:attribute name="label" type="xs:string"/> <!-- param -->
+<xs:attribute name="minValue" type="xs:float"/> <!-- param -->
+<xs:attribute name="maxValue" type="xs:float"/> <!-- param -->
+<xs:attribute name="units" type="xs:string"/> <!-- param -->
```

Comment:
Please only allow the units already available in the existing common.xml, ardupilot.xml ... files

Reply:
use #139 as a type for the units :)